pyLDAvis/js/ldavis.v3.0.0.js

```diff
@@ -512,7 +517,7 @@ var LDAvis = function(to_select, data_or_file_name, color1, color2) {
             .attr("x", 0)
             .attr("y", mdsheight + 10 + (8/2)*barguide.height + 5)
             .style("dominant-baseline", "middle")
-            .text("2. relevance(term w | topic t) = \u03BB * p(w | t) + (1 - \u03BB) * p(w | t)/p(w); see Sievert & Shirley (2014)");
+            .text("2. relevance(term w | topic t; \u03BB) = \u03BB * log(p(w | t)) + (1 - \u03BB) * log(p(w | t)/p(w)); see Sievert & Shirley (2014)");
```

Comment:
Why is this change here?

`relevance(term w | topic t; \u03BB)`

Reply:
The notation is to indicate that Lambda is a parameter of the distribution, such that the formula aligns better to the one in the linked paper:
<img width="245" alt="image" src="https://github.com/bmabey/pyLDAvis/assets/22890124/2b3d6948-a1fa-4364-8781-529b94f250b4">